course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -83,6 +84,7 @@
                         roleName: $scope.getRoleName(user.role_id),
                     });
                 });
+            $scope.searchInProgress = false;
```

Comment:
Nope, I was just thinking that wherever we know we're done with network calls, we set `$scope.searchInProgress = false`.  Using `$q` might not work...the GET only happens if the POST succeeds, right?